app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -127,9 +127,9 @@ $salaried-teacher-training$
 
 Your teacher training course might be provided by: 
 
-* a university (sometimes referred to as university-led training) 
-* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
-* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)  
+* a university, sometimes referred to as university-led training
+* a school or group of schools, sometimes referred to as school-led training, or an apprenticeship 
+* a [charitable organisation such as Teach First](https://www.teachfirst.org.uk/)  
```

Comment:
Your teacher training course may also be provided by a company or charity. For example, [Teach First is a charitable organisation that provides teacher training](https://www.teachfirst.org.uk/).